Web/Scotty.hs

```diff
@@ -89,6 +90,11 @@ defaultHandler = Trans.defaultHandler
 middleware :: Middleware -> ScottyM ()
 middleware = Trans.middleware
 
+-- | Set global size limit for the request body. Requests with body size exceeding the limit will not be
+-- processed and an HTTP error code will be returned to the client. 
+maxRequestBodySize :: Kilobytes -> ScottyM ()
+maxRequestBodySize i = assert (i > 0) $ Trans.routeOptions $ mkRouteOptions i
+
```

Comment:
Do violating requests result in a 500 response or some kinda of 4xx response? It should give a 413 if I'm not mistaken. In other words, do we guarantee that the exception is caught and handled appropriately up the call chain?

Reply:
Maybe also add in the docs that this will crash the server on startup if the argument is not positive.

Reply:
Could we also re-export `Kilobytes`? Ideally, we want people to have a complete picture of the API after importing only `Web.Scotty` and no other modules.

Reply:
Many of the functions in `Web.Scotty` are monomorphised instantiations of functions in `Web.Scotty.Trans`. Can we do this with this function, please? I'm sure `maxRequestBodySize` would be of interest to people using just the `Web.Scotty.Trans`.

Reply:
> ~Do violating requests result in a 500 response or some kinda of 4xx response? It should give a 413 if I'm not mistaken.~ (Answered here: https://github.com/scotty-web/scotty/pull/278/files#r646170275)

I should have been more specific in the docs, that is my mistake

Reply:
> Maybe also add in the docs that this will crash the server on startup if the argument is not positive.

I did, thanks for pointing it out

Reply:
> Could we also re-export `Kilobytes`? Ideally, we want people to have a complete picture of the API after importing only `Web.Scotty` (or `Web.Scotty.Trans`) and no other modules.

done

Reply:
> Many of the functions in `Web.Scotty` are monomorphised instantiations of functions in `Web.Scotty.Trans`. Can we do this with this function, please? I'm sure `maxRequestBodySize` would be of interest to people using just the `Web.Scotty.Trans`.

done